docs/release_notes/v1.9.4.md

```diff
@@ -0,0 +1,21 @@
+# Dapr 1.9.4
+ 
+## Fixes Mqtt pubsub component failure to read messages on startup when multiple subscriptions are configured
+ 
+### Problem
+ 
+When using the Mqtt component users who had multiple subcriptions could encounter an issue where messages would stop being processed for the component.
+ 
+### Impact
+ 
+This issue only occured for the Mqtt pubsub component and only occured in a specific scenario.
+
+The issue only occurs when there are two subscriptions and messages are only being published to the 1st topic and the subscriber gets killed after publishing a few messages. If the subscriber comes back online and there has been a few (>2) messages published already then it wil only deliver the 1st pending message and stop.
+ 
+### Root cause
+ 
+When dapr tries to subscribe to the 2nd topic, it will try to reset the subscription and call m.consumer.Disconnect where it would halt.
```

Comment:
This is too in-detail for the audience, as it's going into the nitty-gritty details of the Dapr source code. You can keep it higher-level such as:

```suggestion
The issue occurs when there are two or more subscriptions, messages are being published to the first topic only, and the subscriber gets killed after publishing a few messages. If the subscriber comes back online after a few (two or more) messages have been published already, then it will only deliver the first pending message and stop.

In the situation above, Dapr could enter into a deadlock and stop processing messages from MQTT.

The issue was traced back to an error in the upstream MQTT SDK used by Dapr
```